starter-code/index.html

```diff
@@ -3,40 +3,66 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">  
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <header id="site-header">
+    <img class="brand-logo" src="./images/spotify-logo.png"/>
+    <nav class="navigation">
+      <ul class="nav-list">
+        <li><a href="">Premium</a></li>
+        <li><a href="">Discover</a></li>
+        <li><a href="">Help</a></li>
+        <li><a href="">Download</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main id="site-main">
+    <div class="main-teaser">
+      <div class="teaser-heading">
+        <h1>Music for everyone.</h1>
+        <p>Spotify is now free on mobile, tablet and computer.<br> Listen to the right music, wherever you are.</p>
+      </div>
+    </div>
+    <div class="main-content">
+      <h2>What’s on Spotify?</h2>
+      <hr>
+      <div class="content-info">
+        <div class="content-music">
```

Comment:
You can reuse styling code here! Better set the same class for all the divs that look the same.